src/v/cluster/cloud_metadata/offsets_recovery_rpc_types.h

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2023 Redpanda Data, Inc.
+ *
+ * Licensed as a Redpanda Enterprise file under the Redpanda Community
+ * License (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * https://github.com/redpanda-data/redpanda/blob/master/licenses/rcl.md
+ */
+#pragma once
+
+#include "cluster/cloud_metadata/error_outcome.h"
+#include "cluster/errc.h"
+#include "model/fundamental.h"
+#include "model/metadata.h"
+#include "model/timeout_clock.h"
+#include "serde/envelope.h"
+#include "ssx/sformat.h"
+#include "utils/fragmented_vector.h"
```

Comment:
Did you mean to use `fragmented_vector` in these structs? Otherwise this include is unused.

Reply:
Done, I originally wanted to use `fragmented_vector` when this class contained everything in `cluster/cloud_metadata/offsets_snapshot.h`. Removed